webapp/graphql/Fragments.js

```diff
@@ -62,6 +62,29 @@ export const postFragment = gql`
   }
 `
 
+export const groupFragment = gql`
+  fragment group on Group {
+    id
+    groupName: name
+    slug
+    disabled
+    deleted
+    about
+    description
+    descriptionExcerpt
+    groupType
+    actionRadius
+    categories {
+      id
+      slug
+      name
+      icon
+    }
+    locationName
+    myRole
```

Comment:
In my opinion we need the `location` here, because it's used in the group teaser:

```suggestion
    locationName
    location {
      name: name${lang}
    }
    myRole
```

If I try it appears on the group teaser!
Therefore we need the `i18n` or `lang`.